internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -724,6 +769,70 @@ func flattenSqlVirtualMachineAutoPatching(autoPatching *sqlvirtualmachine.AutoPa
 		},
 	}
 }
+	
+func expandSqlVirtualMachineAssessmentSettings(input []interface{}) *sqlvirtualmachine.AssessmentSettings {
+	if len(input) == 0 {
+		return nil
+	}
+	assessmentSetting := input[0].(map[string]interface{})
+	
+	if len(assessmentSetting["schedule"]) == 0 {
+		schedule_data := nil
+	}
+	else {
+		schedule_data := &sqlvirtualmachine.Schedule{
+			Enable:                        utils.Bool(true),
+			WeeklyInterval:                utils.Int32(int32(autoPatchingSetting["weekly_interval"].(int))),
+			MonthlyOccurrence:             utils.Int32(int32(autoPatchingSetting["monthly_occurrence"].(int))),
+			DayOfWeek:                     sqlvirtualmachine.DayOfWeek(autoPatchingSetting["day_of_week"].(string)),
+			StartTime:                     autoPatchingSetting["start_time"]
+		}
+	}
+	return &sqlvirtualmachine.AssessmentSettings{
+		Enable:                        utils.Bool(true),
+		RunImmediately:                utils.Bool(assessmentSetting["run_immediately"].(bool)),
+		schedule:                      schedule_data,
+	}
+}
+	
+func flattenSqlVirtualMachineAssessmentSettings(assessmentSettings *sqlvirtualmachine.AssessmentSettings) []interface{} {
```

Comment:
This is missing the invocation in the `resourceMsSqlVirtualMachineRead`

Reply:
@myc2h6o Ok I just updated it thanks